examples/comet/comet-pytorch.ipynb

```diff
@@ -0,0 +1,439 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b8954812",
+   "metadata": {},
+   "source": [
+    "# Track and Compare Models with Comet"
```

Comment:
If this is meant to just be a general comet example that happens to use pytorch then let's name the file just `comet-example.ipynb`